pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -64,12 +64,59 @@ func newGitHubScraper(
 	}
 }
 
-func getNumPrPages(
+func getNumBranchPages(
+	ghs *githubScraper,
+	ctx context.Context,
+	client graphql.Client,
+	repoName string,
+	now pcommon.Timestamp,
+) (int, error) {
+	branchCount, err := ghs.getBranchCount(ctx, client, repoName, ghs.cfg.GitHubOrg)
+	if err != nil {
+		ghs.logger.Sugar().Errorf("error getting branch count", zap.Error(err))
+		return 0, err
+	}
+	ghs.logger.Sugar().Debugf("branch count: %v for repo %v", branchCount, repoName)
```

Comment:
This might be pedantic but I feel like we should reword these messages to something like:

```suggestion
	ghs.logger.Sugar().Debugf("%v repo has %v branches", repoName, branchCount)
```

or


```suggestion
	ghs.logger.Sugar().Debugf("repo: %v; branch count: %v", repoName, branchCount)
```

The current phrasing seems odd when I read it.